lib/webview-helpers.js

```diff
@@ -405,11 +405,12 @@ helpers.getWebviewDetails = function getWebviewDetails (adb, webview) {
  * See https://chromedriver.chromium.org/capabilities for more details.
  */
 helpers.createChromedriverCaps = function createChromedriverCaps (opts, deviceId) {
-  const caps = {
-    chromeOptions: {
-      androidPackage: opts.chromeOptions?.androidPackage || opts.appPackage,
-    }
-  };
+  const caps = { chromeOptions: {} };
+
+  const androidPackage = opts.chromeOptions?.androidPackage || opts.appPackage;
+  if (androidPackage) {
+    caps.chromeOptions.androidPackage = androidPackage;
```

Comment:
it probably makes sense to add a comment here to explain why androidPackage  should not be set to null